benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,278 @@
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --code scripts/custom_components.py
+        --paths.tables ${vars.tables_path}
+
+  - name: "init-labels"
+    help: "Initialize labels first before training"
+    script:
+      - >-
+        python -m spacy init labels
+        configs/${vars.ner_config}.cfg
+        corpus/labels/${vars.dataset}.labels
+        --verbose 
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --paths.tables tables/${vars.dataset}/${vars.dataset}-train.tables
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+
+  - name: "train"
+    help: "Train NER model."
+    script:
+      - >-
+        python -m spacy train
+        configs/${vars.ner_config}.cfg
+        --nlp.lang ${vars.language}
+        --output training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --paths.vectors ${vars.vectors}
+        --paths.tables ${vars.tables_path}/${vars.dataset}/${vars.dataset}-train.tables
+        --components.tok2vec.model.embed.include_static_vectors ${vars.include_static_vectors}
+        --nlp.batch_size ${vars.batch_size}
+        --system.seed ${vars.seed}
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+    deps:
+      - corpus/${vars.dataset}-train.spacy
+      - corpus/${vars.dataset}-dev.spacy
+    outputs:
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-last
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+
+  - name: "train-adjust-rows"
+    help: "Train NER model with adjustable number of rows."
+    script:
+      - >-
+        python -m spacy train
+        configs/${vars.ner_config}.cfg
+        --nlp.lang ${vars.language}
+        --output training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --paths.vectors ${vars.vectors}
+        --paths.tables ${vars.tables_path}/${vars.dataset}/${vars.dataset}-train.tables
+        --components.tok2vec.model.embed.include_static_vectors ${vars.include_static_vectors}
+        --components.tok2vec.model.embed.rows '${vars.rows}'
+        --nlp.batch_size ${vars.batch_size}
+        --system.seed ${vars.seed}
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+    deps:
+      - corpus/${vars.dataset}-train.spacy
+      - corpus/${vars.dataset}-dev.spacy
+    outputs:
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-last
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+
+  - name: "train-hash"
```

Comment:
This also doesn't run:
```
=========================== Initializing pipeline ===========================
✘ Config validation error
tok2vec.model.embed -> num_hashes       extra fields not permitted
{'@architectures': 'spacy.MultiHashEmbed.v2', 'width': 256, 'attrs': ['NORM', 'PREFIX', 'SUFFIX', 'SHAPE'], 'rows': [5000, 2500, 2500, 2500], 'include_static_vectors': True, 'num_hashes': 4}
```

Reply:
This one only runs if you use the `multifewerhashembed.cfg`.

Reply:
Also added an error message if you don't provide `multifewerhashembed`.

Reply:
Yes, nice to add the custom error & warning in the docs!